packages/core/src/indexing/build-indexing.ts

```diff
@@ -2,6 +2,7 @@ import { type DatabaseIndexApi, PostgresIndexApi, SqliteIndexApi } from './datab
 import knex from 'knex'
 import { DiagnosticsLogger, Networks } from '@ceramicnetwork/common'
 import * as fs from 'fs'
+import { ISyncQueryApi } from '../sync/interfaces.js'
```

Comment:
What is this used for?

Reply:
Unused, should probably make that an error in lint. Originally was plumbing through the dependency.